back/app/services/folders_finder_admin_service.rb

```diff
@@ -0,0 +1,45 @@
+class FoldersFinderAdminService
+  def self.execute(scope, params)
+    folders = filter_status(scope, params)
+    folders = filter_folder_manager(folders, params)
+    folders = search(folders, params)
+
+    folders
+      .order('project_folders_folders.created_at DESC, project_folders_folders.id ASC')
+  end
+
+  # FILTERING METHODS
+  def self.filter_status(scope, params = {})
+    status = params[:status] || []
+    return scope if status.blank?
+
+    scope
+      .joins("INNER JOIN admin_publications ON admin_publications.publication_id = project_folders_folders.id AND admin_publications.publication_type = 'ProjectFolders::Folder'")
+      .where(admin_publications: { publication_status: status })
```

Comment:
Was there a reason for using the SQL join type? Think something like this would be clearer maybe
```suggestion
    scope
     .joins(:admin_publications)
     .where(admin_publications: { publication_type: 'ProjectFolders::Folder', publication_status: status })
```